### PR TITLE
Update python v3.11

### PR DESCRIPTION
**Pull request by langmtt**

### CODE REVIEW

File: templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml [hunk 1]
```diff
@@ -131,6 +131,7 @@ Resources:
             reason: No KMS Key Provided
     Condition: cCreateCloudWatchLogGroup
     Properties:
+      KmsKeyId: !Ref AWS::NoValue
```
Comment (himanshu219) [inline templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml:134]: why do we have no value for kms key id ? Is it kmskey id automatically generated by AWS?

File: templates/apps/sumo-aws-apps/guardduty-org/guardduty-org-configuration.yaml [hunk 2]
```diff
@@ -294,14 +311,14 @@ Resources:
         - 'CAPABILITY_AUTO_EXPAND'
       StackInstancesGroup:
         - Regions:
-            -  !Select [0, !Split [",", !Ref EnabledRegions]]
+            - !Ref 'AWS::Region'
```
Comment (himanshu219) [inline templates/apps/sumo-aws-apps/guardduty-org/guardduty-org-configuration.yaml:314]: Do we need to deploy this stackset only in one region ? not in all guardduty enabled regions

**Review by himanshu219 — APPROVED** [review]
LGTM

File: templates/apps/sumo-aws-apps/sumologic-app-utils/src/sumologic.py [hunk 3]
```diff
@@ -72,7 +72,8 @@ def post(self, method, params, headers=None, version=DEFAULT_VERSION):
 
     def put(self, method, params, headers=None, version=DEFAULT_VERSION):
         endpoint = self.get_versioned_endpoint(version)
-        r = self.session.put(endpoint + method, data=json.dumps(params), headers=headers)
+        r = self.session.put(endpoint + method, data=json.dumps(params), headers=headers,timeout=None)
```
Comment (himanshu219) [inline templates/apps/sumo-aws-apps/sumologic-app-utils/src/sumologic.py]: lets put some value for timeout may be 2 min

File: templates/apps/sumo-aws-apps/guardduty-org/guardduty-org-configuration.yaml (hunk 2 above)
Comment (langmtt) [reply templates/apps/sumo-aws-apps/guardduty-org/guardduty-org-configuration.yaml:314]: Role it deploy only one region

File: templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml (hunk 1 above)
Comment (langmtt) [reply templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml:134]: LogGroup don't need KMS

File: templates/apps/sumo-aws-apps/sumologic-app-utils/src/sumologic.py (hunk 3 above)
Comment (langmtt) [reply templates/apps/sumo-aws-apps/sumologic-app-utils/src/sumologic.py]: Updated

**Pull request merged**